eos-tech-support/eos-download-image

```diff
@@ -273,6 +284,9 @@ class EosDownloadImage(object):
         if args.product == 'fnde' and args.personality == 'base':
             args.personality = 'fnde_aluno'
 
+        if args.url.startswith(INTERNAL_BASE_URL):
```

Comment:
What if `args.url` is `None` because `--url` was not specified?

```console
$ git checkout T20033-add-url-and-size
Branch T20033-add-url-and-size set up to track remote branch T20033-add-url-and-size from origin.
Switched to a new branch 'T20033-add-url-and-size'
  tensionsheet ~/src/endlessm/eos-meta
$ eos-tech-support/eos-download-image 
Traceback (most recent call last):
  File "eos-tech-support/eos-download-image", line 431, in <module>
    main()
  File "eos-tech-support/eos-download-image", line 426, in main
    app = EosDownloadImage(session)
  File "eos-tech-support/eos-download-image", line 287, in __init__
    if args.url.startswith(INTERNAL_BASE_URL):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

I think this check should be something like:

```python
if args.url is not None and args.url.startswith(INTERNAL_BASE_URL):
```